oap-server/server-bootstrap/pom.xml

```diff
@@ -280,6 +280,7 @@
                         <exclude>fetcher-prom-rules/</exclude>
                         <exclude>envoy-metrics-rules/</exclude>
                         <exclude>meter-analyzer-config/</exclude>
+                        <exclude>openapi-definitions/</exclude>
```

Comment:
I think `exclude` is not enough.

> org.apache.skywalking.oap.server.library.module.ModuleStartException: path not found: openapi-definitions

This error makes the booting fails. You should modify these two to create an empty folder at least.

https://github.com/apache/skywalking/blob/4e7d80fb88a603e076f6c53ed9817d81dc55da3d/apm-dist/src/main/assembly/binary.xml#L53-L69
https://github.com/apache/skywalking/blob/4e7d80fb88a603e076f6c53ed9817d81dc55da3d/apm-dist-es7/src/main/assembly/binary-es7.xml#L53-L69

Reply:
I have seen these but i wasn't sure if this would create empty folder. 
So do you think `<include>openapi-definitions/*</include>` would fix that? I'm going to check that before pushing.

Reply:
I think this could fix. Although the including file(in the bootstrap module) is only a demo

Reply:
I have finally build whole distribution and confirmed this creates folder with demo service. Also i was reluctant if this include affects server-bootstrap jar files but it does not. So thanks for the feedback